akka-docs/src/test/scala/docs/stream/operators/SourceFromFuture.scala

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.{Done, NotUsed}
+import akka.stream.scaladsl._
```

Comment:
It would be useful to users to also include the most important imports in the code snipped